tests/test_markdown.py

```diff
@@ -125,3 +125,18 @@ async def test_load_non_existing_file() -> None:
             await pilot.app.query_one(Markdown).load(
                 Path("---this-does-not-exist---.it.is.not.a.md")
             )
+async def test_anchor_links() -> None:
```

Comment:
This is looking good (I'm looking to pull this in, in part as a fix for #3094), but I don't think I understand what the test is trying to do here; especially given that it fails.

What is it aiming to test exactly?

Reply:
Sorry I committed by error I didn't know the right way to test it.

Reply:
That's fine, I just wanted to check first. I'm happy to look at adding any appropriate testing. 🙂